canonical/vanilla-framework
# Update references to repo location in the new org

## Done

Updates any references of repo URL to new location in Canonical org.

## QA

- Open [demo](https://vanilla-framework-4525.demos.haus/)
- Review [home page](https://vanilla-framework-4525.demos.haus/), make sure all links to GH are to correct org
- Review [contribute page](https://vanilla-framework-4525.demos.haus/contribute), make sure contributors are correctly displayed from API
